### DISCUSSION TITLE
Bridging transaction done on Rari through stargate is not coming to blockscout.

### DISCUSSION THREAD

**Discussion by Sadham-Hussian** [blockscout/blockscout#10911] [General]
https://stargate.finance/bridge

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.

**Reply by Sadham-Hussian** [reply]
Got it, thanks.

Got the required details via token transfers
